Cryokinesis and Geladikinesis mutations

Time-Green opened the pull request:
Adds cryokinesis in the spirit of goon. It allows you to shoot a beam that freezes people and makes floors slippery.
Adds geladikinesis, gives you the ability to make snow and snow-structures. 

Also fixed snow walls runtiming like fucking crazy because they're treated like objects in crafting

Also added snow tiles to snow crafting

:cl:
add: Adds the cryokinesis and geladikinesis mutations
fix: Snow walls no longer runtime everything
/:cl:
geladi means frozen. I swear im not a frozen fan bit nixkinesis doesnt sound that great
Why: Cryokinesis is fun to mess around with, and geladikinesis is great if you want a bunch of assistants turning dorms into Siberia

Time-Green committed:
Adds cryokinesis and renames the cold resistence file to space adaptation

Time-Green committed:
adds cryokinesis

ATH1909 commented on code/datums/mutations/cold.dm:
```diff
@@ -0,0 +1,41 @@
+/datum/mutation/human/geladikinesis
+	name = "Geladikinetic Creation"
+	desc = "Allows the user to concentrated moisture and sub-zero forces into snow."
```
Replaced "concentrated" with "concentrate".

ATH1909 commented:
Wait, is this a stealth temperature gun buff (or nerf? is the new way the cold beams work less or more effective than the old way?)?

Hey, I'm not complaining, but you should probably mention that in the PR.

Time-Green committed:
Merge remote-tracking branch 'upstream/master' into more-mutations

Time-Green committed:
fixes icon conflict

is there actually a proper way to fix them thats not copying master and reimplimenting your own sprites?

Time-Green committed:
fixes a typo and lowers instability of cryokines by 5

Cyberboss commented on code/__DEFINES/flags.dm:
```diff
@@ -1,162 +1,162 @@
-
-//alternate appearance flags
-#define AA_TARGET_SEE_APPEARANCE (1<<0)
+/*
```
Revert the line ending changes

Time-Green replied:
how would I do that? I've reset the file to master and nothing changed

Cyberboss replied:
https://stackoverflow.com/a/11341700/3976486

Cyberboss commented on code/modules/projectiles/projectile/special/temperature.dm:
```diff
@@ -16,3 +16,14 @@
+
+/obj/item/projectile/temp/cryo/on_range()
+	var/turf/T = get_turf(src)
+	if(istype(T, /turf/open))
```
```suggestion
	if(isopenturf(T))
```

Cyberboss commented on code/modules/spells/spell_types/conjure.dm:
```diff
@@ -80,9 +80,10 @@
-	if (item && !QDELETED(item))
+	if (delete_old && item && !QDELETED(item))
 		qdel(item)
 		item = null
```
```suggestion
```

Cyberboss commented on code/modules/spells/spell_types/conjure.dm:
```diff
@@ -80,9 +80,10 @@
 /obj/effect/proc_holder/spell/targeted/conjure_item/cast(list/targets, mob/user = usr)
-	if (item && !QDELETED(item))
+	if (delete_old && item && !QDELETED(item))
 		qdel(item)
```
```suggestion
		QDEL_NULL(item)
```

Cyberboss commented on code/game/objects/items/stacks/stack.dm:
```diff
@@ -191,9 +191,15 @@
 		var/obj/O
 		if(R.max_res_amount > 1) //Is it a stack?
 			O = new R.result_type(usr.drop_location(), R.res_amount * multiplier)
+		else if(ispath(R.result_type, /turf/))
```
```suggestion
		else if(ispath(R.result_type, /turf))
```

Cyberboss committed:
Apply suggestions from code review

Co-Authored-By: Time-Green <timkoster1@hotmail.com>

Time-Green committed:
maybe reverts line ending?

Time-Green committed:
unfucks flags?

Time-Green committed:
probably unfucks line endings

i swear to fucking god why does it even do this. i never even touched the file and reverting it to master doesnt do anything. fuck you git

Time-Green commented:
Thanks. I unfucked it
